lib/orator/index.js

```diff
@@ -327,23 +327,34 @@ class Orator {
           avatarURL
         } = response
 
-        const hook = await findOrCreateManagedWebhook(
-          msg.channel,
-          bot.user.id,
-          this.webhookName
-        )
+        try {
+          const hook = await findOrCreateManagedWebhook(
+            msg.channel,
+            bot.user.id,
+            this.webhookName
+          )
 
-        newMessage = await bot.executeWebhook(hook.id, hook.token, {
-          ...content,
-          username,
-          avatarURL,
-          wait: true
-        })
+          newMessage = await bot.executeWebhook(hook.id, hook.token, {
+            ...content,
+            username,
+            avatarURL,
+            wait: true
+          })
+        } catch (error) {
+          switch (error.message) {
+            default:
```

Comment:
default case should throw, check the message do the webhook error response thing if it is a permission error

Reply:
sorry, default case should probably send a different message to the user

Reply:
Looking at this, we don't need to send messages from inside the error handler, lets make a new variable that we can use in place of `content` after using `content` for the webhook, we can call it `responseContent` this way the switch statement can just set that variable and we can send the response message if needed (if `responseContent` is not falsy) this will help clean up